bin/ref_arch_setup

```diff
@@ -33,37 +37,170 @@ BANNER
   end
 end
 
-subcommands = {
+def install_global_opts(opt)
+  opt.on("--user <username>", "Username for bolt ssh to target host") do |value|
+    @options["user"] = value
+  end
+  opt.on("--password <password>", "Password for bolt ssh to target host") do |value|
+    @options["password"] = value
+  end
+  opt.on("--private-key <path>", "Path to private key file for bolt ssh to target host") do |value|
+    @options["private_key"] = value
+  end
+end
+
+def install_generate_opts(opt)
+  opt.on("--primary-master <hostname>", "Hostname of primary master") do |value|
+    @options["primary_master"] = value
+  end
+  opt.on("--pe-version <version>", "PE version for base pe.conf file") do |value|
+    @options["pe_version"] = value
+  end
+end
+
+def install_bootstrap(opt)
+  # TODO: remove primary master once we implement getting the host from the pe.conf
+  opt.on("--primary-master <hostname>", "Hostname of primary master") do |value|
+    @options["primary_master"] = value
+  end
+  opt.on("--pe-tarball <path>", "Path to PE tarball") do |value|
+    @options["pe_tarball"] = value
+  end
+  opt.on("--pe-version <version>", "PE version to be installed") do |value|
+    @options["pe_version"] = value
+  end
+  opt.on("--pe-conf <path>", "Path to pe.conf file") do |value|
+    @options["pe_conf"] = value
+  end
+end
+
+def install_infra_agent_install(opt)
+  opt.on("--pe-conf <path>", "Path to pe.conf file") do |value|
+    @options["pe_conf"] = value
+  end
+end
+
+def install_configure(opt)
+  opt.on("--pe-conf <path>", "Path to pe.conf file") do |value|
+    @options["pe_conf"] = value
+  end
+end
+
+def install_opts(opt)
+  install_global_opts(opt)
+  # This one is here because it is required for generate-pe-conf but not necessarily install
+  opt.on("--console-password <password>", "Password for the PE console") do |value|
+    @options["console_password"] = value
+  end
+  install_generate_opts(opt)
+  install_bootstrap(opt)
+  install_infra_agent_install(opt)
+  install_configure(opt)
+end
+
+commands = {
   "install" => OptionParser.new do |opt|
-    opt.banner = "Usage: ref_arch_setup install [options]"
-    opt.on("--target-host value", "Host to install PE on") do |value|
-      options["target_host"] = value
-    end
-    opt.on("--pe-tarball-path value", "Path to PE tarball") do |value|
-      options["pe_tarball_path"] = value
-    end
-    opt.on("--pe-conf-path value", "Path to pe.conf file") do |value|
-      options["pe_conf_path"] = value
+    opt.banner = <<-BANNER
```

Comment:
The option summary "--console-password <password>" forces the description onto the next line. Setting the summary width to at least 33 resolves the issue (we may want to add extra padding):
`opt.set_summary_width(33)`